CHANGELOG.rst

```diff
@@ -1,6 +1,33 @@
 Changelog
 =========
 
+1.0.0 (2021-02-xx)
```

Comment:
Should preface the bullets with a sentence like "After 8 years, we are finally pleased to announce Arrow v1.0. Thanks to the entire Python community for helping to make Arrow the amazing package it is today!